### PR TITLE
#16 Display X axis categorical values

### PR DESCRIPTION
**Pull request by AstridBer**
This PR contributes to issue #16

It displays the categories on the x-axis. 

X-axis

### CODE REVIEW

**Comment by MajaLinder** [comment]
image
Have you tried with more categories or long names? How do we handle when the categories don't fit/have enough space?

**Comment by AstridBer** [comment]
That is a good point! I am not sure how to handle that. Maybe it is best to work on it in a new issue?

**Comment by MajaLinder** [comment]
Yes, according to the acceptance criteria we can merge this, and look at the categories overlapping later. Good work with the x-axis!

File: src/utility/methods.js [hunk 1]
```diff
@@ -46,6 +46,24 @@
...
 }
+
+
+/**
+ * Render a scale label
+ * @param {number} value
+ */
+ export function createLabelX(value) {
+    let label = createDiv("scale-labelX", "" + value);
+    /*
```
[the remark this answers is not in the document]
Comment (np9693) [reply src/utility/methods.js:57]: The code commented here is left out for reference for future development, especially to understand CSS styling.

**Review by MajaLinder — APPROVED** [review]

**Pull request merged**